### ISSUE TITLE
mirage-seal breaks with upper-case build directory

### ISSUE THREAD

**Issue by senier** [mirage/mirage#434]
I'm trying to build a mirage-based HTTPS server on Debian/x86_64 following the instructions in [1]. When running mirage-seal from a directory that starts (?) with an uppercase letter, it bails out with an exception claiming that the directory is invalid.

# How to reproduce: #

1. Create a build directory starting (?) with an uppercase letter
    `$ cd; mkdir A`

2. Create files necessary to run mirage-seal
    ```
$ cd ~/A
$ mkdir -p secrets html
$ echo TEST > html/index.html
$ selfsign -c secrets/server.pem -k secrets/server.key -d 365 example.com
    ```

3. Run mirage-seal from the build directory
   `$ mirage-seal --data html --keys secrets -t unix`

# Observed Error #
 ```
     => rm -rf /tmp/mirage-seal
     => mkdir -p /tmp/mirage-seal
     => mkdir -p /tmp/mirage-seal/keys/tls
     exec-dir: /tmp/mirage-seal
     => cp secrets/server.pem /tmp/mirage-seal/keys/tls
     => cp secrets/server.key /tmp/mirage-seal/keys/tls
     => cd /tmp/mirage-seal && SEAL_DATA=/home/alex/A/html SEAL_DHCP=true SEAL_HTTPS=1 SEAL_KEYS=/tmp/mirage-seal/keys mirage configure -t unix
    Mirage      Using scanned config file: config.ml
    Mirage      Processing: /tmp/mirage-seal/config.ml
    Mirage      => rm -rf /tmp/mirage-seal/_build/config.*
    Mirage      => cd /tmp/mirage-seal && ocamlbuild -use-ocamlfind -tags annot,bin_annot -pkg mirage config.cmxs
    [ERROR]  /home/alex/a/html is not a valid directory.
    mirage-seal: internal error, uncaught exception:
                 Failure("cd /tmp/mirage-seal && SEAL_DATA=/home/alex/A/html SEAL_DHCP=true SEAL_HTTPS=1 SEAL_KEYS=/tmp/mirage-seal/keys mirage configure -t unix: failed with exit code 1.")
```

# Expected behavior #

Build the image as with a lower-case directory:

```
$ cd ~
$ mkdir a
$ cd a
$ mkdir -p secrets html
$ echo TEST > html/index.html
$ selfsign -c secrets/server.pem -k secrets/server.key -d 365 example.com
$ mirage-seal --data html --keys secrets -t unix
```
(build process)

```
To run your sealed unikernel, use `sudo ./mir-seal`
```

# Environment #

* Debian Linux (testing), x86_64
* mirage-seal set up as in [2]

[1] https://mirage.io/blog/mirage-seal
[2] https://github.com/mirage/mirage-seal/blob/master/README.md

**Comment by lnmx** [comment]
Thanks for the thorough bug report.  The issue is specific to mirage-seal, I've entered an issue there: mirage/mirage-seal#18.

**Comment by samoht** [comment]
This is fixed by a new release of mirage-seal (0.4.3)
